worker/uniter/runner/runner_test.go

```diff
@@ -59,7 +59,7 @@ exit 42
 	c.Assert(err, jc.ErrorIsNil)
 
 	c.Assert(result.Code, gc.Equals, 42)
-	c.Assert(strings.ReplaceAll(string(result.Stdout), "\n", ""), gc.Equals, paths.GetCharmDir()+"BAR")
+	c.Assert(strings.ReplaceAll(string(result.Stdout), "\n", ""), gc.Equals, paths.GetCharmDir())
```

Comment:
So we need to change this test to use a KUBERNETES_FOO env var, not just delete the check :-)